compiler/rustc_lint/messages.ftl

```diff
@@ -415,6 +415,16 @@ lint_improper_ctypes_union_layout_help = consider adding a `#[repr(C)]` or `#[re
 lint_improper_ctypes_union_layout_reason = this union has unspecified layout
 lint_improper_ctypes_union_non_exhaustive = this union is non-exhaustive
 
+lint_incompatible_target_modifiers =
+    mixing `{$flag_name_suffixed}` will cause an ABI mismatch
+    .note1 = `{$flag_name_suffixed}`=`{$flag_local_value}` in crate `{$local_crate}`, `{$flag_name_suffixed}`=`{$flag_extern_value}` in crate `{$extern_crate}`
```

Comment:
```suggestion
    .note1 = `{$flag_name_suffixed}={$flag_local_value}` in this crate is incompatible with `{$flag_name_suffixed}={$flag_extern_value}` in dependency `{$extern_crate}`
```

Likewise with the other comment, `-Zregparm=` rather than `regparm` is better, this should look as similar to what the user will have passed to the compiler as possible.